integration_tests/run-all.sh

```diff
@@ -131,12 +131,12 @@ if [[ $TESTS_TO_RUN =~ "all" || $TESTS_TO_RUN =~ "light" ]]; then
   CLI_SHELL_TESTS=$(find integration_tests/light_client_proxy -type f -name '*.sh' -not -name "common.sh")
 
```

Comment:
Please add integration tests for query certs by subject-key-id (CLI, REST, gRPS)

Reply:
Please loot at `integration_tests/grpc_rest/pki/helpers.go:754`